### ISSUE TITLE
Does not hook onto Fabled Skyblock

### ISSUE THREAD

**Issue by Michael-Amaya** [mastercake10/CustomOreGen#26]
Get the following error running CustomOreGen with TacoSpigot@f8ba67d6 with Skyblock plugin FabledSkyblock Build 79.2.

Get the following error:
```
27.09 04:25:37 [Server] INFO [CustomOreGen] You are not using any skyblock plugin! This plugin only works with a listed skyblock plugin! (check documentations)
27.09 04:25:37 [Server] INFO [CustomOreGen] Disabling CustomOreGen v1.3.17-SNAPSHOT
27.09 04:25:37 [Server] ERROR Error occurred while disabling CustomOreGen v1.3.17-SNAPSHOT (Is it up to date?)
```

Seems to be an issue hooking into Fabled Skyblock, running the latest version.

**Comment by mastercake10** [comment]
> Get the following error:

That's not an error, it just indicates that you should use a skyblock plugin from the list.

Nevertheless, I'll leave this issue open in case anyone wants to add support for this plugin.

**Comment by Michael-Amaya** [comment]
I see, I thought it would support it since there’s a hook called FabledSkyblock in the hooks folder. I’m not a Java Dev, but I can definitely look into it, I just want to know what the purpose of that hook is.

**Comment by mastercake10** [comment]
fixed in 849d23e.
